js_modules/dagit/packages/ui/src/components/TagSelector.tsx

```diff
@@ -130,11 +145,12 @@ export const TagSelector = ({
   }, [
     allTags,
     dropdownStyles,
+    items,
     renderDropdown,
     renderDropdownItem,
-    rowHeight,
     selectedTags,
     setSelectedTags,
+    totalHeight,
```

Comment:
This looks good to me - I think @hellendag has the context on why `react-virtualized` vs `@tanstack/react-virtual`. I wonder if an alternative to the bugs we've been seeing is upgrading to a newer version of react-virtualized (maybe?)